README.md

```diff
@@ -130,7 +130,8 @@ to `./runner.yml` and overriding relevant properties.
 
 This command will also:
 
-- Symlink the theme in  `./build/modules/custom/oe_authentication` so that it's available for the test site
+- **Please note:** project files and directories are symlinked within the test site by using the
```

Comment:
I suppose it would be better to use the same description from https://github.com/openeuropa/open_vocabularies/pull/28/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R45.